Makefile

```diff
@@ -33,7 +33,7 @@ install:
 other_build:
 	GOARCH=amd64 GOOS=darwin go build --ldflags "${.LD_FLAGS}" -o bin/aem.darwin ./cmd/aem
 	GOARCH=amd64 GOOS=linux go build --ldflags "${.LD_FLAGS}" -o bin/aem.linux ./cmd/aem
-	GOARCH=amd64 GOOS=windows go build --ldflags "${.LD_FLAGS}" -o bin/aem.exe ./cmd/aem
+	GOARCH=amd64 GOOS=windows go build -tags timetzdata --ldflags "${.LD_FLAGS}" -o bin/aem.exe ./cmd/aem
```

Comment:
why only for windows?

Reply:
and there is another place to cover for that - https://github.com/wttech/aemc/blob/main/.goreleaser.yaml#L23

Reply:
> why only for windows?

Only windows lacks the tzdata library



> and there is another place to cover for that - https://github.com/wttech/aemc/blob/main/.goreleaser.yaml#L23

Done

Reply:
Instead of adding a separate build id.. Hmm maybe just that flag will be ignored when on non-windows OS will be built

Reply:
okay, adding this tag for all archs won't hurt

Reply:
so align it as well in Makefile ;)